frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -15,59 +15,27 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
-import { getStatus, TStatus } from '../../../api/bitbox02';
+import { getStatus } from '../../../api/bitbox02';
 import { statusChanged } from '../../../api/devicessync';
-import { UnsubscribeList, unsubscribe } from '../../../utils/subscriptions';
+import { useSync } from '../../../hooks/api';
 import { BB02Settings } from '../../settings/bb02-settings';
 
-type Props = {
+type TProps = {
   deviceID: string;
   deviceIDs: string[];
   hasAccounts: boolean;
 }
 
-type State = {
-    status: '' | TStatus;
-}
-
-export class BitBox02 extends Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = {
-      status: '',
-    };
-  }
+export const BitBox02 = ({ deviceID, deviceIDs, hasAccounts }: TProps) => {
+  const status = useSync(
+    () => getStatus(deviceID),
+    cb => statusChanged(deviceID, () => {
```

Comment:
@thisconnect found a way to use `useSync` anyway, which looks better as it contains the mounted ref stuff too. PTAL.